packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -148,57 +181,57 @@ export const styles = css`
         ButtonAppearance.outline,
         css`
             .control {
-                background-color: transparent;
                 border-color: rgba(${actionRgbPartialColor}, 0.3);
             }
 
             .control:hover {
                 background-color: transparent;
-                border-color: ${borderHoverColor};
             }
 
             .control${focusVisible} {
                 background-color: transparent;
-                border-color: ${borderHoverColor};
             }
 
             .control:active {
+                outline: none;
+            }
+
+            .control::before:active {
                 background-color: ${fillSelectedColor};
-                border-color: ${fillSelectedColor};
+                outline: none;
+                padding: 1px;
             }
 
             :host([disabled]) .control {
+                border-color: rgba(${borderRgbPartialColor}, 0.3);
+            }
```

Comment:
The selection target of this element is `.control` and should be grouped with the other `.control` targetted selectors.